Kentico.AcceleratedMobilePages/AmpFilter.cs

```diff
@@ -360,5 +360,22 @@ private void RemoveAttribute(HtmlDocument doc, string xPath, string attrName)
                 }
             }
         }
+
+        /// <summary>
+        /// Splits the Friendly URL Extension, and returns the first element
+        /// </summary>
+        private string GetFriendlyExtension()
```

Comment:
This method can be "inlined" directly to the setting's property. 
Actually all usages of 
https://github.com/Kentico/kentico-amp/blob/02b78c239bde5f4ffbc86e31030c18ddb65035a8/Kentico.AcceleratedMobilePages/Settings.cs#L12
expect that this setting returns only one extension (which is also promised by name of the property)

Instead this method, you can just change getter of this property to
`SettingsKeyInfoProvider.GetValue(SiteContext.CurrentSiteName + ".CMSFriendlyURLExtension").Split(';')[0];`
and it'll do precisely the same job as this method.

To explain why it is not needed this amount of code, `"".Split(';')` always returns at least the original string, in this case empty string. Therefore it is not needed to check whether `extensions.Length > 0` or the length of extension array after the split is more than 0.